File: src/main/java/scrumweb/dto/fieldcontent/ListElementsContainerContentDto.java
package scrumweb.dto.fieldcontent;

import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import scrumweb.dto.projectfield.ListElementDto;

import java.util.Set;

@NoArgsConstructor
@Getter @Setter
public class ListElementsContainerContentDto extends FieldContentDto {
    private Set<ListElementDto> elements;

    public ListElementsContainerContentDto(Long projectFieldId, String projectFieldName, Set<ListElementDto> elements) {
        super(projectFieldId, projectFieldName);
        this.elements = elements;
    }
}

File: src/frontend/src/app/model/fields-content/ListElementsContainerContentDto.ts
import { ListElementDto } from './ListElementDto';
import { FieldContentDto } from './FieldContentDto';

export class ListElementsContainerContentDto extends FieldContentDto {
    elements: Array<ListElementDto> = [];

    constructor(projectFieldId: number, projectFieldName: string, elements: Array<ListElementDto>) {
        super(projectFieldId, projectFieldName);
        this.elements = elements;
    }
}
